code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -572,6 +572,8 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 			protected_contents = TRUE
 		else
 			shielding_powered = FALSE
+			SSair.start_processing_machine(src)
+			investigate_log("shielding turned off due to power loss")
```

Comment:
is this just for visuals or does it hook to something else

Reply:
it won't start processing on atmos, which is what causes it to actually break, without telling atmos to start processing it

Reply:
ah I see ok